server/backend/api/app/crud/crud_user_role.py

```diff
@@ -34,7 +35,7 @@ def create_user_role(
         crud_utils.raise_http_exception(HTTPStatus.BAD_REQUEST, error_msg)
 
     # Determine if user already exists or add a new user.
-    fam_user = crud_user.find_or_create(db, request.user_type_code, request.user_name)
+    fam_user = crud_user.find_or_create(db, request.user_type_code, request.user_name, user_creator)
```

Comment:
Same comment for the variable naming, and other places.

Reply:
Same as above